Don't try to install bundled glslang as part of ncnn

====> Checking for pkg-plist issues (check-plist)
===> Parsing plist
===> Checking for items in STAGEDIR missing from pkg-plist
Error: Orphaned: include/glslang/Include/BaseTypes.h
Error: Orphaned: include/glslang/Include/Common.h
Error: Orphaned: include/glslang/Include/ConstantUnion.h
Error: Orphaned: include/glslang/Include/InfoSink.h
Error: Orphaned: include/glslang/Include/InitializeGlobals.h
Error: Orphaned: include/glslang/Include/PoolAlloc.h
Error: Orphaned: include/glslang/Include/ResourceLimits.h
Error: Orphaned: include/glslang/Include/ShHandle.h
Error: Orphaned: include/glslang/Include/SpirvIntrinsics.h
Error: Orphaned: include/glslang/Include/Types.h
Error: Orphaned: include/glslang/Include/arrays.h
Error: Orphaned: include/glslang/Include/glslang_c_interface.h
Error: Orphaned: include/glslang/Include/glslang_c_shader_types.h
Error: Orphaned: include/glslang/Include/intermediate.h
Error: Orphaned: include/glslang/MachineIndependent/Initialize.h
Error: Orphaned: include/glslang/MachineIndependent/LiveTraverser.h
Error: Orphaned: include/glslang/MachineIndependent/ParseHelper.h
Error: Orphaned: include/glslang/MachineIndependent/RemoveTree.h
Error: Orphaned: include/glslang/MachineIndependent/Scan.h
Error: Orphaned: include/glslang/MachineIndependent/ScanContext.h
Error: Orphaned: include/glslang/MachineIndependent/SymbolTable.h
Error: Orphaned: include/glslang/MachineIndependent/Versions.h
Error: Orphaned: include/glslang/MachineIndependent/attribute.h
Error: Orphaned: include/glslang/MachineIndependent/gl_types.h
Error: Orphaned: include/glslang/MachineIndependent/glslang_tab.cpp.h
Error: Orphaned: include/glslang/MachineIndependent/iomapper.h
Error: Orphaned: include/glslang/MachineIndependent/localintermediate.h
Error: Orphaned: include/glslang/MachineIndependent/parseVersions.h
Error: Orphaned: include/glslang/MachineIndependent/preprocessor/PpContext.h
Error: Orphaned: include/glslang/MachineIndependent/preprocessor/PpTokens.h
Error: Orphaned: include/glslang/MachineIndependent/propagateNoContraction.h
Error: Orphaned: include/glslang/MachineIndependent/reflection.h
Error: Orphaned: include/glslang/Public/ShaderLang.h
Error: Orphaned: include/glslang/SPIRV/GLSL.ext.AMD.h
Error: Orphaned: include/glslang/SPIRV/GLSL.ext.EXT.h
Error: Orphaned: include/glslang/SPIRV/GLSL.ext.KHR.h
Error: Orphaned: include/glslang/SPIRV/GLSL.ext.NV.h
Error: Orphaned: include/glslang/SPIRV/GLSL.std.450.h
Error: Orphaned: include/glslang/SPIRV/GlslangToSpv.h
Error: Orphaned: include/glslang/SPIRV/Logger.h
Error: Orphaned: include/glslang/SPIRV/NonSemanticDebugPrintf.h
Error: Orphaned: include/glslang/SPIRV/SPVRemapper.h
Error: Orphaned: include/glslang/SPIRV/SpvBuilder.h
Error: Orphaned: include/glslang/SPIRV/SpvTools.h
Error: Orphaned: include/glslang/SPIRV/bitutils.h
Error: Orphaned: include/glslang/SPIRV/disassemble.h
Error: Orphaned: include/glslang/SPIRV/doc.h
Error: Orphaned: include/glslang/SPIRV/hex_float.h
Error: Orphaned: include/glslang/SPIRV/spirv.hpp
Error: Orphaned: include/glslang/SPIRV/spvIR.h
Error: Orphaned: include/glslang/build_info.h
Error: Orphaned: lib/cmake/OGLCompilerTargets-%%CMAKE_BUILD_TYPE%%.cmake
Error: Orphaned: lib/cmake/OGLCompilerTargets.cmake
Error: Orphaned: lib/cmake/OSDependentTargets-%%CMAKE_BUILD_TYPE%%.cmake
Error: Orphaned: lib/cmake/OSDependentTargets.cmake
Error: Orphaned: lib/cmake/SPIRVTargets-%%CMAKE_BUILD_TYPE%%.cmake
Error: Orphaned: lib/cmake/SPIRVTargets.cmake
Error: Orphaned: lib/cmake/glslangTargets-%%CMAKE_BUILD_TYPE%%.cmake
Error: Orphaned: lib/cmake/glslangTargets.cmake
Error: Orphaned: lib/libGenericCodeGen.a
Error: Orphaned: lib/libMachineIndependent.a
Error: Orphaned: lib/libOGLCompiler.a
Error: Orphaned: lib/libOSDependent.a
Error: Orphaned: lib/libSPIRV.a
Error: Orphaned: lib/libglslang.a

--- CMakeLists.txt.orig	2022-07-28 09:47:27 UTC
+++ CMakeLists.txt
@@ -513,7 +513,7 @@ if(NCNN_VULKAN)
             if(NCNN_SHARED_LIB)
                 option(SKIP_GLSLANG_INSTALL "" ON)
             endif()
-            add_subdirectory(glslang)
+            add_subdirectory(glslang EXCLUDE_FROM_ALL)
             if(NCNN_SHARED_LIB)
                 if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR (CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND NOT CMAKE_CXX_COMPILER_FRONTEND_VARIANT MATCHES "MSVC"))
                     target_compile_options(glslang PRIVATE -fvisibility=hidden -fvisibility-inlines-hidden)
